commit dad4b510eec547fee48661d0af1d2d1c579be335
Author: Joe Perches <joe@perches.com>
Date:   Wed Nov 10 15:46:18 2010 -0800

    fs/udf: Add printf format/argument verification
    
    Add __attribute__((format... to udf_warning.
    
    All arguments matched formats, no other changes necessary.
    
    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Jan Kara <jack@suse.cz>

--- a/fs/udf/udfdecl.h
+++ b/fs/udf/udfdecl.h
@@ -111,6 +111,8 @@ struct extent_position {
 };
 
 /* super.c */
+
+__attribute__((format(printf, 3, 4)))
 extern void udf_warning(struct super_block *, const char *, const char *, ...);
 static inline void udf_updated_lvid(struct super_block *sb)
 {
